client/testitems/birds.html

```diff
@@ -27,7 +27,7 @@
 
         <div id="container"></div>
```

Comment:
just a preference if you're working on two different features such as the conversion and load modal, it is better to split them into the branches so in case one has conflicts, the good one can be merged.

Reply:
Yeah, that was because I forgot to switch branches and would lose my work if I didn't submit under this branch.